infrastructure/000-aws-base/vpc.tf

```diff
@@ -67,6 +67,10 @@ module "us_east_1_public_subnet_cidrs" {
   networks = [for az_name in sort(data.aws_availability_zones.us_east_1.names) : {name = az_name, new_bits = 4}]
 }
 
+locals {
```

Comment:
`locals` for terraform is just a block of locally used variables. It can be nice to pre-calculate some values that aren't tied to resource creation since it's not easy to reference the fields of a resource from other fields of the resource.